src/libcore/ops/function.rs

```diff
@@ -127,27 +145,35 @@ pub trait FnMut<Args> : FnOnce<Args> {
     extern "rust-call" fn call_mut(&mut self, args: Args) -> Self::Output;
 }
 
-/// A version of the call operator that takes a by-value receiver.
+/// The version of the call operator that takes a by-value receiver.
+///
+/// Instances `FnOnce` can be called, but might not be callable multiple times.
```

Comment:
The reason for phrasing it this way is that any instance of e,g, `Fn` is also an instance of `FnOnce`, so an instance of `Fn`, while being an instance of `FnOnce`, can be called multiple times. That's why I added the second sentence about when the only known thing is `FnOnce`.